lib/ansible/cli/doc.py

```diff
@@ -1179,27 +1248,17 @@ def get_man_text(doc, collection_name='', plugin_type=''):
         pad = display.columns * 0.20
         limit = max(display.columns - int(pad), 70)
 
-        plugin_name = doc.get(context.CLIARGS['type'], doc.get('name')) or doc.get('plugin_type') or plugin_type
-        if collection_name:
-            plugin_name = '%s.%s' % (collection_name, plugin_name)
-
-        text.append("> %s    (%s)\n" % (plugin_name.upper(), doc.pop('filename')))
+        text.append("> %s %s (%s)\n" % (plugin_type.upper(), _format(doc.pop('plugin_name'), 'bold'), doc.pop('filename')))
 
         if isinstance(doc['description'], list):
             desc = " ".join(doc.pop('description'))
         else:
             desc = doc.pop('description')
 
-        text.append("%s\n" % textwrap.fill(DocCLI.tty_ify(desc), limit, initial_indent=opt_indent,
-                                           subsequent_indent=opt_indent))
-
-        if 'version_added' in doc:
-            version_added = doc.pop('version_added')
-            version_added_collection = doc.pop('version_added_collection', None)
-            text.append("ADDED IN: %s\n" % DocCLI._format_version_added(version_added, version_added_collection))
+        text.append(" %s\n" % textwrap.fill(DocCLI.tty_ify(desc), limit, initial_indent=opt_indent, subsequent_indent=opt_indent))
```

Comment:
There is an extra space for the first line of the description
```suggestion
        text.append("%s\n" % textwrap.fill(DocCLI.tty_ify(desc), limit, initial_indent=opt_indent, subsequent_indent=opt_indent))
```

Reply:
he, had not seen this but already made similar change